src/analysis/analysis-model.js

```diff
@@ -18,6 +26,14 @@ module.exports = Model.extend({
     this._initBinds();
   },
 
+  url: function () {
+    var url = this.get('url');
+    if (this.get('apiKey')) {
+      url += '?api_key=' + this.get('apiKey');
```

Comment:
I propose to provide the query strings through the backbone expected data prop on options arg to fetch instead of building the full URL yourself, e.g.:

```js
fetch: function(options) {
  options = options || {};
  options.data = options.data || {};
  
  var apiKey = this.get('apiKey');
  if (apiKey) {
    options.data.api_key = apiKey;
  }
  
  return Model.prototype.fetch.call(this, options);
}
```